includes/wikia/ExternalUser_Wikia.php

```
@@ -292,6 +301,15 @@ protected function addToDatabase( User &$User, $password, $email, $realname ) {
 			$ret = false;
 		}
 
+        catch ( ExternalUserException $e ) {
+            \Wikia\Logger\WikiaLogger::instance()->info(
```

Why not report an error if we're catching exception here?